textattack/attack_args.py

```diff
@@ -371,7 +372,9 @@ def create_loggers_from_args(cls, args):
             if not os.path.exists(dir_path):
                 os.makedirs(os.path.dirname(txt_file_path))
 
-            attack_log_manager.add_output_file(txt_file_path)
+            # attack_log_manager.add_output_file(txt_file_path)
```

Comment:
I think you can remove this commented out code

Reply:
This has been fixed